ssl/t1_trce.c

```diff
@@ -1286,7 +1286,7 @@ static int ssl_print_certificate(BIO *bio, int indent,
     BIO_printf(bio, "ASN.1Cert, length=%d", (int)clen);
     x = d2i_X509(NULL, &q, clen);
     if (!x)
-        BIO_puts(bio, "<UNPARSEABLE CERTIFICATE>\n");
+        BIO_puts(bio, "<UNPARSABLE CERTIFICATE>\n");
```

Comment:
@t8m Do you think we should leave this one alone, in case someone is checking for the error?

Reply:
It is tracing output, so I'm ambivalent. But yeah, reverting this is the safer option.

Reply:
Reverted